git-coco

```diff
@@ -291,6 +291,10 @@ prlist() {
         author="$(echo $author | awk -F '/' '{print $NF}')"
         dst_branch="$(echo "${dst_branch/refs\/heads\/}")"
         src_branch="$(echo "${src_branch/refs\/heads\/}")"
+        
+        if [ "${#title}" -gt 60 ]; then
+            title="$(echo "${title}" | cut -c 1-57)..."
```

Comment:
You could probably use bash string manipulation to do this without `cut`, since nothing else in the script uses `cut` it might make sense.

Reply:
fixed